apps/studio/src/lib/db/clients/sqlserver.js

```diff
@@ -118,7 +118,8 @@ function buildFilterString(filters) {
   return filterString
 }
 
-function genSelectOld(table, offset, limit, orderBy, filters, schema) {
+function genSelectOld(table, offset, limit, orderBy, filters, schema, selects) {
+  const selectString = selects.join(', ')
```

Comment:
The column names need escaping here - `selects.map((s) => wrapIdentifier(s)).join(", ")`